src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -650,6 +650,107 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_batch_norm_relu_global_sta
     this->add_matcher(m, callback);
 }
 
+void ngraph::runtime::cpu::pass::CPUFusion::construct_batch_norm_infer_relu_with_multiply_add()
```

Comment:
Please add comment on what the transformation does here. An example would do.

Reply:
Added comments.